src/registrar/tests/test_models_domain.py

```diff
@@ -523,7 +526,7 @@ def test_update_is_unsuccessful(self):
         raise
 
 
-class TestRegistrantNameservers(TestCase):
+class TestRegistrantNameservers(MockEppLib):
```

Comment:
@neilmb Mind looking over what we are testing here and let us know if there are other cases we should unit test

Reply:
@abroddrick @dave-kennedy-ecs I like that this fills in all of the skipped tests, that seems like a great start. The last thing I think that I want to see is a round-trip or end-to-end test of the nameserver management view in the Django domain management application. I think that Dave's work should roll up to there so that we should be able to load `/domain/<id>/nameservers`, enter some change in the form, press `save`, and then check that the EPP calls were made and that the page now shows the new nameservers.

If that isn't all hooked up yet, then maybe make a skipped test in `src/registrar/tests/test_views.py` that describes doing that and that test can be filled in when the connections are made?